.gitignore

```diff
@@ -1 +1 @@
-build/
+/gopath/
```

Comment:
I found it cleaner that way. If you want, we can make the three files explicit as
```
/gopath/
!/gopath/*/.keep
!/gopath/src/github.com/holocm/holo
```